cmd/crane/main.go

```diff
@@ -27,9 +28,18 @@ var rootCmd = &cobra.Command{
 	Run:   func(cmd *cobra.Command, _ []string) { cmd.Usage() },
 }
 
+var gendoc = flag.String("gendoc", "", "If set, path to generate docs and exit")
+
 func main() {
+	flag.Parse()
```

Comment:
It's weird that we have two flag parsers.

Reply:
Yeah I agree. I thought about promoting it to a command, but then it's weird because users shouldn't have to run it or ever think about it. So I wanted to hide it from generated usage and docs. `flag.Parse` seemed like a good place to hide it.

Another option would be to have a _separate_ command that does nothing but generate docs for `crane`, which would require `rootCmd` to become exported, and bleh I just gave up and did the simplest dumbest thing.

Reply:
Exporting rootCmd would actually be *really* nice for any hypothetical tool that might want to extend `crane` with additional features/commands... ;-)

Reply:
Started towards this in #142.